src/messages/worker.rs

```diff
@@ -62,6 +63,8 @@ pub enum ToWorkerMessage {
     CancelTasks(TaskIdsMsg),
     NewWorker(NewWorkerMsg),
     RegisterSubworker(SubworkerDefinition),
+    EnableHardwareMonitoring, //Enables addition of hw_usage information to overview
```

Comment:
I don't see the point of these messages currently. Having separate enable + disable messages makes sense if the worker will sample the hardware state e.g. every one second, and then it would return the most currently sampled state in `GetOverviewResponse`. But in the current implementation, the state is sampled only if `GetOverview` is called anyway, so it would be much simpler to just include a flag like `include_hw_state` in the `Overview` request.

Reply:
I did it, then I lost it while rebasing and it skipped my mind after that